diesel_derives/tests/queryable.rs

```diff
@@ -87,3 +87,16 @@ fn multiple_tables() {
         data
     );
 }
+
+#[test]
+fn name_conflict() {
+    type Field = i32;
+    type Record = i32;
```

Comment:
Probably this is should be `Row`?

Reply:
That would have introduced a test for the `Row` case, so yes that would have been the optimal solution, but it's not that bad to not have it.